site/content/docs/v1.11.0/start-contributing.md

```diff
@@ -16,5 +16,8 @@ Thanks for taking the time to join our community and start contributing!
 [2]: {{< param github_url >}}/blob/main/CONTRIBUTING.md
 [3]: {{< param github_url >}}/issues
 [4]: {{< param slack_url >}}
-[5]: https://github.com/projectcontour/contour-operator/blob/main/docs/CONTRIBUTING.md
-[6]: https://github.com/projectcontour/contour-operator/issues
+[5]: {{< param github_operator_url >}}/blob/main/docs/CONTRIBUTING.md
+[6]: {{< param github_operator_url >}}/issues
+[7]: https://vmware.zoom.us/j/347232187
+[8]: https://hackmd.io/84Xbl4WBTpm7OBhaOAsSiw
+[9]: https://www.youtube.com/playlist?list=PL7bmigfV0EqTBsPrnCkzhu0R4SAWnBjLj
```

Comment:
Honestly doesn't matter but links 7-9 aren't being referenced, the places where they would be referenced still have the URLs directly. Since the `main` page does use the footnote numbers, going fwd we're good, so I'm fine with it as-is.

Reply:
Heh I guess I'm not even sure what issue we're fixing in this PR, just moving links to footnotes?

Reply:
yeah I think another PR fixed the original goal of this PR, so after rebasing there's not much left. :sigh: